### PR TITLE
Add a guzzle notice

### PR DESCRIPTION
**Pull request by jnweiger**
Does that make sense in the known issues section?
Or should it be else where?

### CODE REVIEW

File: modules/ROOT/pages/server_release_notes.adoc [hunk 1]
```diff
@@ -93,7 +93,12 @@ Find below a list of updated apps in comparison with the 10.10.0 complete bundle
 
 === Known Issues
 
-Currently there are no known issues with ownCloud Server 10.11.0. This section will be updated if issues are discovered.
+==== PHP guzzle dependencies
+This release uses guzzle 7. The complete bundle contain several app updates that were migrated from guzzle 5 to guzzle 7. Please check any non-standard apps for guzzle 7 compatibility. If in doubt, disable these apps before upgrading.
```
Comment (EParzefall) [inline modules/ROOT/pages/server_release_notes.adoc:97]:
```suggestion
This release uses guzzle 7. The complete bundle contains several app updates that were migrated from guzzle 5 to guzzle 7. Please check any non-standard apps for guzzle 7 compatibility. If in doubt, disable these apps before upgrading.
```

File: modules/ROOT/pages/server_release_notes.adoc [hunk 2]
```diff
@@ -93,7 +93,12 @@ Find below a list of updated apps in comparison with the 10.10.0 complete bundle
...
+
+Affected third party apps include:
+* richdcuments 2.7.0
```
Comment (EParzefall) [inline modules/ROOT/pages/server_release_notes.adoc:100]:
```suggestion
* richdocuments 2.7.0
```

**Review by EParzefall — APPROVED** [review]

File: modules/ROOT/pages/server_release_notes.adoc [hunk 3]
```diff
@@ -93,7 +93,12 @@ Find below a list of updated apps in comparison with the 10.10.0 complete bundle
 
 === Known Issues
 
-Currently there are no known issues with ownCloud Server 10.11.0. This section will be updated if issues are discovered.
+==== PHP guzzle dependencies
+This release uses guzzle 7. The complete bundle contains several app updates that were migrated from guzzle 5 to guzzle 7. Please check any non-standard apps for guzzle 7 compatibility. If in doubt, disable these apps before upgrading.
+
+Affected third party apps include:
```
Comment (EParzefall) [inline modules/ROOT/pages/server_release_notes.adoc:99]:
```suggestion
Affected third-party apps include:
```

**Review by EParzefall — APPROVED** [review]

File: modules/ROOT/pages/server_release_notes.adoc (hunk 3 above)
Comment (EParzefall) [reply modules/ROOT/pages/server_release_notes.adoc:99]: Sorry for the noise, overlooked a missing dash.

File: modules/ROOT/pages/server_release_notes.adoc [hunk 4]
```diff
@@ -93,7 +93,12 @@ Find below a list of updated apps in comparison with the 10.10.0 complete bundle
...
+* richdocuments 2.7.0
+* onlyoffice 7.5.3
```
[the remark this answers is not in the document]
Comment (jnweiger) [reply modules/ROOT/pages/server_release_notes.adoc]: feel free to improve the wording. "Currently known incompatible apps include" maybe?

[the remark this answers is not in the document]
Comment (EParzefall) [reply modules/ROOT/pages/server_release_notes.adoc]: Sounds good to me. Will adjust the text accordingly.

File: modules/ROOT/pages/server_release_notes.adoc [hunk 5]
```diff
@@ -93,7 +93,12 @@ Find below a list of updated apps in comparison with the 10.10.0 complete bundle
 
 === Known Issues
 
-Currently there are no known issues with ownCloud Server 10.11.0. This section will be updated if issues are discovered.
+==== PHP guzzle dependencies
+This release uses guzzle 7. The complete bundle contains several app updates that were migrated from guzzle 5 to guzzle 7. Please check any non-standard apps for guzzle 7 compatibility. If in doubt, disable these apps before upgrading.
+
+Affected third-party apps include:
```
Comment (EParzefall) [inline modules/ROOT/pages/server_release_notes.adoc:99]:
```suggestion
Currently known incompatible apps include:
```

**Review by EParzefall — APPROVED** [review]

File: modules/ROOT/pages/server_release_notes.adoc [hunk 6]
```diff
@@ -93,7 +93,12 @@ Find below a list of updated apps in comparison with the 10.10.0 complete bundle
 
 === Known Issues
 
-Currently there are no known issues with ownCloud Server 10.11.0. This section will be updated if issues are discovered.
+==== PHP guzzle dependencies
+This release uses guzzle 7. The complete bundle contains several app updates that were migrated from guzzle 5 to guzzle 7. Please check any non-standard apps for guzzle 7 compatibility. If in doubt, disable these apps before upgrading.
```
Comment (mmattel) [inline modules/ROOT/pages/server_release_notes.adoc:97]:
```suggestion
This release uses guzzle 7. The complete bundle contains several app updates that were migrated from guzzle 5 to guzzle 7. When using the minimum bundle, those apps can be downloaded from the marketplacte. Please check any non-standard apps for guzzle 7 compatibility. If in doubt, disable these apps before upgrading.
```

**Pull request merged**